### PR TITLE
SIEM - Dev siem vpn disable bug no vpn gateway

### PR DESCRIPTION
**Pull request by zachreborn**
Bug was identified with enable_vpn_peering = false where the VPN gateway was failing due to propagating_vpgws not having an element

### CODE REVIEW

File: modules/thinkstack/siem/main.tf [hunk 1]
```diff
@@ -68,7 +68,7 @@ resource "aws_nat_gateway" "natgw" {
 ###########################
 
 resource "aws_route_table" "public_route_table" {
-  propagating_vgws = [aws_vpn_gateway.vpn_gateway[0].id]
+  propagating_vgws = var.enable_vpn_peering ? [aws_vpn_gateway.vpn_gateway[0].id] : []
```
Comment (zachreborn) [inline modules/thinkstack/siem/main.tf:71]: Sets the argument to be empty if enable_vpn_peering is false

File: modules/thinkstack/siem/main.tf [hunk 2]
```diff
@@ -81,7 +81,7 @@ resource "aws_route" "public_default_route" {
 
 resource "aws_route_table" "private_route_table" {
   count            = length(var.azs)
-  propagating_vgws = [aws_vpn_gateway.vpn_gateway[0].id]
+  propagating_vgws = var.enable_vpn_peering ? [aws_vpn_gateway.vpn_gateway[0].id] : []
```
Comment (zachreborn) [inline modules/thinkstack/siem/main.tf:84]: Sets the argument to be empty if enable_vpn_peering is false

**Comment by zachreborn** [comment]
Working pull request with the module updates
https://github.com/thinkstack-co/thinkstack_dev_security/pull/1
https://app.terraform.io/app/thinkstack-co/thinkstack_dev_security/runs/run-xGg7nPBZaJxpjfUn

**Review by TheJalpa — APPROVED** [review]
Changes look good, approved!

**Pull request merged**